packages/cadl-ranch-specs/http/parameters/collection-format/main.tsp

```diff
@@ -47,7 +47,7 @@ namespace Header {
   @scenarioDoc("""
 This test is testing sending a csv collection format array header parameters
 """)
-  @route("/csvHeader")
+  @route("/csv")
```

Comment:
@timotheeguerin is there any way that we can validate every route in the main tsp is covered in mockapi.ts? Right now we check based off of scenario name, could we add a step verifying the routes match?